ibu-imager/postpivot/postpivot.go

```diff
@@ -366,3 +370,20 @@ func (p *PostPivot) setNewClusterID(ctx context.Context, client runtimeclient.Cl
 	}
 	return nil
 }
+
+func (p *PostPivot) applyNMStateFiles(seedReconfiguration *clusterconfig_api.SeedReconfiguration) error {
+	if len(seedReconfiguration.RawNMStateConfigs) == 0 {
+		return nil
+	}
+	nmFile := path.Join(p.workingDir, "nmstate.yaml")
+	p.log.Infof("Applying nmstate config files")
+	for _, nmstate := range seedReconfiguration.RawNMStateConfigs {
+		// TODO: we can verify files by running nmstatectl apply --no-commit ?
```

Comment:
I think that would be nice, and we can just log it in case of failure

Reply:
We should not just log, we should fail the process as for me